packages/esm-patient-search-app/src/patient-search-page/advanced-patient-search.component.tsx

```diff
@@ -109,6 +109,13 @@ const AdvancedPatientSearchComponent: React.FC<AdvancedPatientSearchProps> = ({
           }
         }
 
+        if (
+          !patient.extension ||
+          (patient.extension.name !== 'deceased-patient-tag' && patient.extension.name !== 'active-visit-tag')
+        ) {
+          return false;
+        }
+
```

Comment:
You shouldn't need this.

Reply:
@Jexsie Thanks for the review! I have fixed the proposed changes.